sources/packages/web/src/services/http/dto/ProgramYear.dto.ts

```diff
@@ -1,4 +1,4 @@
-export interface ProgramYear {
+export interface ProgramYearAPIOutDTO {
```

Comment:
API in/out DTOs are classes on `API` side. only then it will show up in swagger. like 
![image](https://user-images.githubusercontent.com/77353155/208499976-0db5d93f-a0f0-4ae1-b36f-9d750aaad478.png)